lisa/nic.py

```diff
@@ -456,3 +515,57 @@ def _get_default_nic(self) -> None:
         ).is_true()
         self.default_nic: str = default_interface_name
         self.default_nic_route = dev_match.group()
+
+    def get_used_module(self) -> str:
+        lspci = self._node.tools[Lspci]
+        pci_devices = lspci.get_devices_by_type(
+            constants.DEVICE_TYPE_SRIOV, force_run=True
+        )
+        # there will not be multiple Mellanox types in one VM
```

Comment:
What if the first nic is synthetic only, and the second attaches a pci nic? We know some vm sizes allow limited pci devices. It may happen. So it's better to go through all information to find used modules.